src/diskquota_utility.c

```diff
@@ -1299,12 +1299,19 @@ worker_spi_get_extension_version(int *major, int *minor)
  */
 
 List *
-get_rel_oid_list(void)
+get_rel_oid_list(bool is_init)
 {
 	List *oidlist = NIL;
 	int   ret;
 
-	ret = SPI_execute_with_args("select oid from pg_class where oid >= $1 and (relkind='r' or relkind='m')", 1,
+#define SELECT_FROM_PG_CATALOG_PG_CLASS \
+	"select oid from pg_catalog.pg_class where oid >= $1 and (relkind='r' or relkind='m')"
+#define SELECT_FROM_DISKQUOTA_TABLE_SIZE "select tableid from diskquota.table_size where segid = -1"
+
+	ret = SPI_execute_with_args(is_init ? SELECT_FROM_PG_CATALOG_PG_CLASS
+	                                    " union distinct " SELECT_FROM_DISKQUOTA_TABLE_SIZE
+	                                    : SELECT_FROM_PG_CATALOG_PG_CLASS,
+	                            1,
```

Comment:
is it possible to do this without macros? maybe do something like this?
```c
const char* sql;
if(is_init){
	sql = "...";
}else{
	sql = "...";
}

ret = SPI_execute_with_args(sql,...);
```

Reply:
It is done to avoid copy/paste

Reply:
ok, but let's try to make the code a little more readable. What do you think about such changes?
```suggestion
#define SELECT_FROM_PG_CATALOG_PG_CLASS \
	"select oid from pg_catalog.pg_class where oid >= $1 and (relkind='r' or relkind='m')"
#define SELECT_FROM_DISKQUOTA_TABLE_SIZE \
	"select tableid from diskquota.table_size where segid = -1"

	ret = SPI_execute_with_args(is_init ? SELECT_FROM_PG_CATALOG_PG_CLASS
	                                      " union distinct " 
	                                      SELECT_FROM_DISKQUOTA_TABLE_SIZE

	                                    : SELECT_FROM_PG_CATALOG_PG_CLASS,
	                            1,
```

Reply:
Are your changes clang 13 formatted?

Reply:
Sorry I forgot about it.